packages/expo-av/src/Audio/Recording.ts

```diff
@@ -119,7 +119,8 @@ export const RECORDING_OPTIONS_PRESET_HIGH_QUALITY: RecordingOptions = {
     bitRate: 128000,
   },
   ios: {
-    extension: '.caf',
+    extension: '.mp4',
```

Comment:
```suggestion
    extension: '.m4a',
```
Suggest this should be consistent with Android since they'll both be mpeg-4 containers?

Reply:
Let's switch to `.mp4` as suggested to obtain platforms unity 👍

Reply:
Are you suggesting switch both to `.mp4` or did you mean both to `.m4a`?
Before this PR its Android `.m4a` and iOS: `.caf`

Reply:
No I would keep `m4a` on android since that is what I tested to work (see the issue). I'm happy to make it `mp4` but do not have the bandwidth to test it.

Reply:
Sorry yeah that was directed at @bbarthec

I think we want them both to be `mp4` or `m4a` but I was unclear if @bbarthec was suggesting making iOS `m4a` to match the existing Android value or change both iOS and Android to `mp4`

Reply:
Sorry guys for this misunderstanding of mine 🙇 
I would be awesome if we could ship the same extension/format on both platforms. As there's `.m4a` on Android we should probably lean towards the same on iOS. Is is possible? If so, I would go with it 😉

Reply:
@bbarthec I have not tested it so can't confirm it works and do not have the time to test.

We can change to `m4a` globally but I can't guarantee it works. If you can test it on 2 hardware devices we can safely do `m4a`. Otherwise using `m4a` and `mp4` should be fine. We currently have 2 extensions anyway.

Reply:
If @mnightingale is on board I can amend the PR to include it.

Reply:
I've used m4a for iOS/Android for quite a while, there'll be no issue mp4 === m4a they are a common extension for the same container (MPEG-4).
`m4a` is commonly used when the file only contains an audio track, likewise `mp4` when it has both audio and video tracks.
Since we are dealing with audio recordings it makes sense to unify on `m4a`.

Per my other comment I think we could consider changing the codecs to AAC_HE and possibly dropping the bitrate.

TL;DR do `.m4a` for both the high and low presets